src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/ReconfigureUpcomingMessage.tsx

```diff
@@ -29,47 +29,45 @@ export default function V2V3ReconfigureUpcomingMessage() {
     // If duration is unset/0, changes take effect immediately to current FC
     message = (
       <Trans>
-        Your project's current funding cycle has no duration. Changes you make
-        below will take effect immediately.
+        Your project's current cycle has no duration. Edits you make below will
+        take effect immediately.
       </Trans>
     )
   } else if (ballotStrategyLength === undefined) {
     message = (
       <Trans>
-        Changes will take effect according to the project's custom ballot
-        contract.
+        The edits you make below may take effect, depending on the project's
+        custom edit deadline contract.
       </Trans>
     )
   } else if (ballotStrategyLength > secondsUntilNextFC) {
     message = (
       <Trans>
-        Changes you make will take effect according to your{' '}
-        <strong>{ballotStrategy.name}</strong> reconfiguration rule (the first
-        funding cycle following{' '}
+        Due to your <strong>{ballotStrategy.name}</strong> contract, edits you
+        make will not take effect until the first funding cycle which starts at
```

Comment:
```suggestion
        make will not take effect until the first funding cycle. The First funding cycle starts at
```

Reply:
I see why this is confusing — fixing, but with a slightly different sentence.